android/build.gradle

```diff
@@ -63,7 +63,9 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.2'
     implementation 'com.amplitude:analytics-connector:1.0.0'
     implementation 'androidx.core:core-ktx:1.7.0'
-    compileOnly 'androidx.compose.ui:ui:1.6.7'
+    implementation 'androidx.activity:activity-ktx:1.9.1'
+    compileOnly 'androidx.fragment:fragment-ktx:1.8.2'
```

Comment:
For compose we check if compose is available or not. For this one, how do we make sure it is available before calling any of the classes inside this class?

Reply:
We're checking if the `FragmentActivity` class is available [here](https://github.com/amplitude/Amplitude-Kotlin/pull/216/files#diff-115c82866176416f03f37c21dea752145c553bba0d00bbc905db9e630af6225bR59) before using it.